File: javascript/javascript/day6_strings.js
// Day 6 - JavaScript Strings

let text = "Hello Kalyani";

// Count vowels
let vowels = "aeiouAEIOU";
let count = 0;
for (let char of text) {
    if (vowels.includes(char)) {
        count++;
    }
}
console.log(`Number of vowels in '${text}':`, count);

// Reverse string
let reverseText = text.split('').reverse().join('');
console.log("Reversed string:", reverseText);

File: python/python/day6_strings.py
# Day 6 - Python Strings

# Count vowels in a string
text = "Hello Kalyani"
vowels = "aeiouAEIOU"
count = 0
for char in text:
    if char in vowels:
        count += 1
print(f"Number of vowels in '{text}':", count)

# Reverse a string
reverse_text = text[::-1]
print(f"Reversed string: {reverse_text}")